contents/handbook/growth/yc-onboarding.md

```diff
@@ -0,0 +1,81 @@
+---
+title: YC Onboarding
+sidebar: Handbook
+showTitle: true
+---
+
+As a YCombinator company ourselves, we want to be supportive of other YC companies that want to use PostHog. As such, our onboarding process for YC companies is different and more extensive than our general process.
+
+## The YC Deal
+
+Our YC Deal is as follows:
+
+One year of Enterprise Cloud with up to 10 million events per month *or* a one-year enterprise license if they want to self-host PostHog.
+
+Companies also get a 50$ gift card to spend on our merch and enterprise support (i.e. a private Slack with our core team members).
+
+For YC W21 companies, we also offer:
+
+- A choice of Airpods or Allbirds shoes
+- A referral scheme to get more of the above
```

Comment:
- We will give more merch (an extra $50 voucher for our store) to the company they refer to us if they make a referral to another current batch company